examples/react/components/Content.tsx

```diff
@@ -130,11 +129,8 @@ const Content: React.FC = () => {
           })
           .catch((err) => {
             try {
-              swal ( "Oops" ,  "" + err ,  "error" )
               //alert("fail " + err);
-            } catch(e) {
-
-            }
+            } catch (e) {}
```

Comment:
This is still a different. Check our `dev` branch and copy how it is.

Reply:
hi @amirsaran3, just fixed this